### ISSUE TITLE
Questions about electrodes

### ISSUE THREAD

**Issue by Nivek92** [UlysseCoteAllard/3DC-Armband#2]
Hey I got 2 questions in regards to the electrodes.

Question 1: What are the dimensions of a single electrode (the board and the radius of the exposed copper)?
Unfortunately I can't work with the altium files because I'm working on Linux and the Altium designer is not available on linux.

Edit: From the stl file I would assume the outer dimensions of a single electrode board are roughly 25mm x 11mm, correct?

Question 2: Will a copper pour on the pcb help reduce noise or could it somehow interfere with the measurements?
I would assume it can't interfere with the signal and instead help reduce noise but I just want to double check.
Could I connect the copper pour to ref or channel or should I just add an unused GND?

Question 3: On the Myo Band they use 3 electrodes per segment. Do they use 2 channel and 1 reference or what is the 3rd electrode? Why do you only use 2 electrodes per segment? What are the advantages and disadvantages?

PS: Sorry for all those rather dumb questions but I have no background in bio signals and am rather new to electronics, so I just want to make sure before I go and print the PCBs.

**Comment by UlysseCoteAllard** [comment]
Hi, just want to say that I will answer very soon. Just need to verify something to make sure I don't answer something wrong. Sorry for the delay.

**Comment by Nivek92** [comment]
No worries. Better safe than sorry.

**Comment by UlysseCoteAllard** [comment]
Hi, Sorry for the delay and thank you for the questions. 

Q1: The diameter of the electrodes is around ~1cm (8.5mm)
Q2: On the electronic PCB, we already have a ground pane, a copper pour there wouldn't change anything. However, a copper pour on the electrodes connected to the reference might help for noise (we do not think it would really help, but maybe). What would probably help would be to shield the signals from the electrodes in a metal linked to the reference, which was not done in the current design. 
Q3: We think that the Myo Armband has a differential amplifier for each pod with two input (+ and -) independent with a reference in the middle. In our design, we also have a differential amplifier where the + is independent and the - was voluntarily linked to the reference. Following tests that we conducted, we concluded that the recording quality was not impacted by this technique and we had about half as many signals to route. 

Hopefully, this can help you! 

Have a great day!
